client/src/pages/ItemList/ItemList.module.scss

```
@@ -0,0 +1,9 @@
+@import "../../global.scss";
+
+.ItemList {
+	display: grid;
+	grid-template-columns: repeat(3, 1fr);
```

Kann man ein wenig responsiver machen mit grid-auto-columns
```suggestion
	grid-auto-columns: minmax(50px, 150px); // Passe die minimum bzw maximum Werte an so wie du es brauchst
```